dev/PagerControl/PagerControlSelectedIndexChangedEventArgs.h

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "PagerControlSelectedIndexChangedEventArgs.g.h"
+
+class PagerControlSelectedIndexChangedEventArgs :
+    public winrt::implementation::PagerControlSelectedIndexChangedEventArgsT<PagerControlSelectedIndexChangedEventArgs>
+{
+public:
+    PagerControlSelectedIndexChangedEventArgs(const int oldIndex,const int newIndex) :
+        m_PreviousPageIndex(oldIndex), m_NewPageIndex(newIndex) {};
```

Comment:
>oldIndex [](start = 28, length = 8)

I think naming the parameter previousIndex makes sense.

Reply:
Good point, updated now.